frontend/src/utils/slug.ts

```diff
@@ -83,11 +85,17 @@ export function sqlReviewPolicySlug(reviewPolicy: SQLReviewPolicy): string {
   return [slug(reviewPolicy.name), reviewPolicy.environment.id].join("-");
 }
 
-export function connectionSlug(database: Database): string {
-  return [
-    slug(database.instance.name),
-    database.instance.id,
-    slug(database.name),
-    database.id,
-  ].join("_");
+export function connectionSlug(
+  instance: Instance,
+  database?: Database
+): string {
+  const parts = [instanceSlug(instance)];
+  if (database && database.id !== UNKNOWN_ID) {
+    parts.push(databaseSlug(database));
+  }
+  return parts.join("_");
```

Comment:
how about double dash "--"

"_" is not a word break

Reply:
What about `/sql-editor/{instance_slug}/{database_slug}` (database_slug is optional). This will be NOT ambiguous to `/sql-editor/sheet/{sheet_slug}`